torchvision/models/detection/__init__.py

```diff
@@ -5,3 +5,15 @@
 from .retinanet import *
 from .ssd import *
 from .ssdlite import *
+
+import warnings
+
+from .. import _BETA_DETECTION_IS_ENABLED
+
+if not _BETA_DETECTION_IS_ENABLED:
```

Comment:
Here we mark the entire `torchvision.models.detection` as Beta and accessing anything within its namespace will throw a warning unless `torchvision.models.enable_beta_detection()` is called.